nucypher/characters/control/specifications.py

```diff
@@ -60,7 +60,7 @@ def validate_response(self, interface_name: str, response: dict) -> bool:
 
 class AliceSpecification(CharacterSpecification):
 
-    __create_policy = (('bob_encrypting_key', 'bob_verifying_key', 'm', 'n', 'label'),  # In
+    __create_policy = (('bob_encrypting_key', 'bob_verifying_key', 'm', 'n', 'label', 'expiration'),  # In
```

Comment:
How can we handle conditionally-required inputs in specifications? In this case, `value` is mandatory when not federated. Seems kind of tricky to do it.

Reply:
Good eye! I've been thinking about this too, I think it's possible to allow values to be passes as `None`  all the way into character, with some modification to specification handling seen here.

Reply:
It might second Issue, but here's one <https://github.com/nucypher/nucypher/issues/1065>